notebook/static/notebook/js/actions.js

```diff
@@ -423,6 +423,14 @@ define(function(require){
                 env.notebook.show_command_palette();
             }
         },
+        'toggle-marks': {
```

Comment:
Based on the PR I just submitted, the action name would probably be `toggle-cell-marked`.